src/signals/incident/components/form/FormField/FormField.tsx

```diff
@@ -59,17 +59,18 @@ export interface FormFieldProps extends Pick<ReactiveFormMeta, PickedProps> {
   meta: FormMeta
   options?: FormOptions
   isFieldSet?: boolean
+  children?: ReactNode
```

Comment:
this should not be optional, a form field always is a parent to some other element right?